overlord/ifacestate/handlers.go

```diff
@@ -655,12 +657,22 @@ func (m *InterfaceManager) doAutoConnect(task *state.Task, _ *tomb.Tomb) error {
 			continue
 		}
 
-		ts, err := ConnectOnInstall(st, chg, task, plug.Snap.Name(), plug.Name, slot.Snap.Name(), slot.Name)
+		ignore, err := findSymmetricAutoconnect(st, plug.Snap.Name(), slot.Snap.Name(), task)
 		if err != nil {
-			task.Logf("cannot auto-connect plug %s to %s: %s", connRef.PlugRef, connRef.SlotRef, err)
+			return err
+		}
+
+		if ignore {
 			continue
```

Comment:
maybe already known but unit tests don't seem to hit this

Reply:
Right. One of the two cases is covered now, I'll see if I can get both (the test is a little bit tricky)